### PR TITLE
Remove unused code

### PR DESCRIPTION
**Pull request by terencechain**
Sunday funday code cleaning continue 🧹 

This PR removes the unused code via Goland inspection🧹

### CODE REVIEW

File: shared/bytesutil/bytes.go [hunk 1]
```diff
@@ -79,14 +79,6 @@ func FromBytes8(x []byte) uint64 {
 // LowerThan returns true if byte slice x is lower than byte slice y. (little-endian format)
```
Comment (prestonvanloon) [inline shared/bytesutil/bytes.go:79]: Need to delete the comments too?

File: shared/bytesutil/bytes.go [hunk 2]
```diff
@@ -186,17 +178,6 @@ func FromBytes48Array(x [][48]byte) [][]byte {
 }
 
 // Xor xors the bytes in x and y and returns the result.
```
Comment (prestonvanloon) [inline shared/bytesutil/bytes.go:180]: Delete comments too?

File: shared/hashutil/hash.go [hunk 3]
```diff
@@ -104,12 +104,6 @@ func HashKeccak256(data []byte) [32]byte {
 
 // RepeatHash applies the sha256 hash function repeatedly
```
Comment (prestonvanloon) [inline shared/hashutil/hash.go:105]: Comments?

File: shared/testutil/log.go [hunk 4]
```diff
@@ -45,25 +44,3 @@ func assertLogs(t *testing.T, hook *test.Hook, want string, flag bool) {
 // WaitForLog waits for the desired string to appear the logs within a
```
Comment (prestonvanloon) [inline shared/testutil/log.go:44]: Comments

**Comment by terencechain** [comment]
> Please clean up godocs for methods that were deleted

oops sorry, just realized using goland's delete button did not delete the comments

**Pull request merged**